astro.config.mjs

```diff
@@ -24,8 +22,10 @@ export default defineConfig({
 		solidJs(),
 		mdx(),
 		sitemap(),
-		prefetch(),
 	],
+	prefetch: {
+		defaultStrategy: "hover",
+	},
```

Comment:
If you wanted, you could remove this entirely, since `hover` is the default behaviour anyways

```suggestion
```